TheSolutionForArithmetics/CalculateArea/Geometry.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CalculateArea
+{
+    public class Geometry
+    {
+        public static double? areaOfCircle(decimal radius)
+        {
+            if(radius <= 0)
+            {
+                Console.WriteLine("Error, radius cannot be zero or below");
+                return null;
+            }
+            decimal area = (decimal)Math.PI * radius * 2;
+            return Convert.ToDouble(area);
+            //throw new NotImplementedException();
```

Comment:
?

Reply:
man arī '???'
(double)Math.PI * radius * 2;

Reply:
//throw new NotImplementedException(); -- kas ir šis? Neizskatās, ka te tam jābūt